bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/handler/DeviceHandler.java

```diff
@@ -366,14 +367,20 @@ public void handleUpdate(FeatureDataDTO featureDataDTO) {
 
                     if (msg.getDeviceId().contains(config.deviceId) && !"unit".equals(entry)) {
                         if (!"[]".equals(valueEntry)) {
+                            logger.trace("Feature: {} Type:{} Entry: {}={}", featureDataDTO.feature, typeEntry, entry,
+                                    valueEntry);
+
                             if (viUnit != null) {
                                 if (!viUnit.isEmpty()) {
                                     msg.setUnit(viUnit);
                                     unit = UNIT_MAP.get(viUnit);
+                                    if (unit == null) {
+                                        logger.warn(
+                                                "Unknown unit. Could not parse the unit: {} of the Feature: {} Please report an issue on github",
```

Comment:
What do you think of this logging? I want the user to be informed that the binding does not know the unit of the feature. And report it to us on github.
There are many different installations with other units that I am not aware of.